dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DOIDataCiteRegisterService.java

```diff
@@ -132,37 +124,33 @@ public String deactivateIdentifier(String identifier, HashMap<String, String> me
                 rc.setStatus("unavailable");
                 retString = client.inactiveDataset(identifier.substring(identifier.indexOf(":") + 1));
             }
-        } catch (IOException io) {
-
+        } catch (IOException ioe) {
+            logger.log(Level.WARNING, "Exception encountered: ", ioe);
         }
         return retString;
     }
 
     public static String getMetadataFromDvObject(String identifier, Map<String, String> metadata, DvObject dvObject) {
 
-        Dataset dataset = null;
-
-        if (dvObject instanceof Dataset) {
-            dataset = (Dataset) dvObject;
-        } else {
-            dataset = (Dataset) dvObject.getOwner();
-        }
+        Dataset dataset = dvObject instanceof Dataset
+            ? (Dataset) dvObject
+            : (Dataset) dvObject.getOwner();
 
         DataCiteMetadataTemplate metadataTemplate = new DataCiteMetadataTemplate();
         metadataTemplate.setIdentifier(identifier.substring(identifier.indexOf(':') + 1));
         metadataTemplate.setCreators(Util.getListFromStr(metadata.get("datacite.creator")));
         metadataTemplate.setAuthors(dataset.getLatestVersion().getDatasetAuthors());
         if (dvObject.isInstanceofDataset()) {
-            metadataTemplate.setDescription(dataset.getLatestVersion().getDescriptionPlainText());
+            String description = dataset.getLatestVersion().getDescriptionPlainText();
+            metadataTemplate.setDescription(description.replaceAll("(&nbsp;)+", " "));
```

Comment:
Maybe we should use `StringEscapeUtils.unescapeHtml()` for potentially more html entities? Or is it special case since I don't exactly remember.

Reply:
Unfortunately we cannot go this way – if `dataset.getLatestVersion().getDescriptionPlainText()` returns `&nbsp;`, `unescapeHtml()` will convert that to `&amp;nbsp;`.

Reply:
![image](https://user-images.githubusercontent.com/22947763/105156670-6d837900-5b0c-11eb-90e3-87a30a84c7dc.png)
According to my attempts it will just convert it to space :P

Reply:
Ok, I've mistakenly used `escapeHTML()`. As the `unescapeHTML()` works correctly and xml uses UTF-8 as default encoding, I've made the change.